contracts/zero-ex/CHANGELOG.json

```diff
@@ -9,6 +9,10 @@
             {
                 "note": "Use new `checkAllowance` flag in LiquidityProviderFeature, TransformERC20Feature, and MetaTransactionsFeature",
                 "pr": 39
+            },
+            {
+                "note": "Convert metatransactions to use `LibSignature`",
+                "pr": 31
             }
```

Comment:
This needs to go in a new version block because it got published (has a timestamp)